baselines/fedstar/README.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Federated Self-training for Semi-supervised Audio Recognition
+url: [dl.acm.org/doi/10.1145/3520128](https://dl.acm.org/doi/10.1145/3520128)
+labels: [Audio Classification, Semi Supervised learning]
+dataset: [Ambient Context, Speech Commands]
+---
+
+# FedStar: Federated Self-training for Semi-supervised Audio Recognition
+
+> Note: If you use this baseline in your work, please remember to cite the original authors of the paper as well as the Flower paper.
+
+****Paper:**** :[dl.acm.org/doi/10.1145/3520128](https://dl.acm.org/doi/10.1145/3520128)
```

Comment:
```suggestion
****Paper:****  [dl.acm.org/doi/10.1145/3520128](https://dl.acm.org/doi/10.1145/3520128)
```

Reply:
In many places in the `README.md` there are two `:` followed, please remove them.

Reply:
Thanks for poinitng out this error, I have remove all double colons.